Makefile

```diff
@@ -14,3 +14,7 @@ coverage:
 lint:
 	# sudo apt-get install python-flake8
 	$(PYTHON) -m flake8 .
+
+.PHONY: install-dev
+install-dev:
+	ln -s $(shell pwd)/txjuju /usr/local/lib/python2.7/dist-packages/txjuju
```

Comment:
See my comment from https://github.com/juju/txjuju/pull/7#discussion_r82972168 as well: it doesn't fail silently right away, but instead of the second go.